packages/utils/src/PromiseUtils.ts

```diff
@@ -1,16 +1,20 @@
 import CanceledPromiseError from './CanceledPromiseError';
 import TimeoutError from './TimeoutError';
 
+interface CancelablePromise<T> extends Promise<T> {
```

Comment:
I would export this interface. The Pending promise change should import the interface from this file instead of redefining it and I think it makes most sense to export it from utils